R/admin-panel.R

```diff
@@ -488,7 +488,7 @@ df_all_results <- function(results_dir) {
   files <- list_results_files(results_dir, full.names = TRUE)
   if (length(files) == 0L) return(data.frame())
   data <- lapply(files, readRDS)
-  data_df <- lapply(data, as.data.frame)
+  data_df <- lapply(data, function(x) as.data.frame(as.list(x)))
```

Comment:
Sorry, I was a bit rushed when I filed the pull request, so I forgot about the documentation, and to run check. I was indeed not aware about the re-ordering of columns in as_tibble.results, because the CSV export never really worked (at least not for some time), and I can't remember of having seen an exported CSV file. The tests failed because as.data.frame had no stringsAsFactors = F argument, and tests checked for character values. At the end, I modified df_all_results as as.data.frame is able to handle arbitrary nestings of data objects and also deals with duplicated columns in an acceptable way, and I also brought back the reordering. The question is only, if we want to deal with duplicated columns this way or if we actually want to fail, as indicator that something is wrong. I would opt for the first, actually, because at the moment of CSV export it might be already too late, so dealing with duplicate columns in a graceful way could allow some fixing in the post-processing stage. Not sure, though.

Don't know if you agree with my rewrite as I am using some dplyr functions so I had to change the dependencies, I got rid of plyr but introduced dplyr and tidyselect. If that is the case, then there is also currently commented version of df_all_results the just with the necessary fixes and the re-ordering. Maybe you want have a look here in my fork first, before I file an "official" pull request?

Cheers,
Klaus